frontend/src/components/AutoCompleteAsync/index.jsx

```diff
@@ -57,13 +57,11 @@ export default function AutoCompleteAsync({
   }, [debouncedValue]);
 
   const onSearch = (searchText) => {
-    if (searchText && searchText != '') {
```

Comment:
No checks needed here since we are already checking debouncedValue everywhere

Reply:
revert this line of code , this will break when we try to update